source/Tests/Playnite.Tests/SearchViewModelTests.cs

```diff
@@ -30,7 +30,6 @@ public void MatchTextFilterTest()
             Assert.IsTrue(SearchViewModel.MatchTextFilter(null, "", false));
 
             // Acronym tests
-            Assert.IsFalse(SearchViewModel.MatchTextFilter("g", "Gears of War", true));

```

Comment:
Removed because it will return true because the "g" will get matched by normal string search